src/main/java/frc/robot/subsystems/LEDSubsystem.java

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.LEDPattern;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class LEDSubsystem extends SubsystemBase {
+  private static final int kPort = 9;
+  private static final int kLength = 120;
+
+  private final AddressableLED m_led;
+  private final AddressableLEDBuffer m_buffer;
```

Comment:
```suggestion
  private final AddressableLED led;
  private final AddressableLEDBuffer buffer;
```

We don't use the `m_` prefix in other places in our codebase.